pages/parent/signup.tsx

```diff
@@ -20,8 +20,6 @@ import { HeardFromPage } from "@components/parent-form/HeardFromPage";
 import { ParentCreatedPage } from "@components/parent-form/ParentCreatedPage";
 import colourTheme from "@styles/colours";
 
-const RADIO_YES = "yes";
-const RADIO_NO = "no";
```

Comment:
I think these might be used for the health form after they are added as discussed below? If not, we can remove them.